src/app/main.c

```diff
@@ -354,6 +354,9 @@ main (int    argc,
 
   /* avoid gvfs (http://bugzilla.gnome.org/show_bug.cgi?id=526454) */
   g_setenv ("GIO_USE_VFS", "local", TRUE);
+  /* There's not really a "root dconf" right now */
+  if (getuid () == 0)
+    g_assert (g_setenv ("GSETTINGS_BACKEND", "memory", TRUE));
```

Comment:
Every time I see this, I forget why it's there and what it does. Can we add a link to the GNOME bug (https://bugzilla.gnome.org/show_bug.cgi?id=767183) and/or a short description of why it's needed here?

The [original PR](https://github.com/projectatomic/rpm-ostree/pull/312) mentions that it was to avoid a proxy warning, but I can't remember what exactly the interaction was. Was it because we didn't want libsoup looking up configured proxy settings?